packages/modules/src/modules/banners/guardianWeekly/GuardianWeeklyBanner.tsx

```diff
@@ -87,8 +89,18 @@ const GuardianWeeklyBanner: React.FC<BannerRenderProps> = ({
     return (
         <section css={banner} data-target={bannerId}>
             <Container>
-                <Columns cssOverrides={columns} collapseBelow="tablet">
-                    <Column width={5 / 12} css={topLeftComponent}>
+                <Columns>
+                    <Column width={1} cssOverrides={iconAndClosePosition}>
+                        <Inline space={1}>
+                            <div css={logoContainer}>
+                                <SvgRoundelDefault />
+                            </div>
+                            <CloseButton onClick={onCloseClick} />
```

Comment:
Given that this is now first in the reading order for screen readers, it might be a good idea to make the button text something like 'Close this banner' rather than just 'Close', to give it a bit more context. Otherwise this looks good!